e2e/cluster/local/scripts/cluster-prepare

```diff
@@ -5,7 +5,7 @@ set -e
 : "${ARCH?required}"
 : "${DIST?required}"
 : "${ALPINE_TAG:=3.14}"
-: "${KUBECTL_TAG:=v1.18.20}"
+: "${KUBECTL_TAG:=v1.21.9}"
 : "${SONOBUOY_TAG:=v0.18.5}"
```

Comment:
Hi Brandon,
I tried upgrading sonobuoy's version to 0.56.2 when I worked on the PR but the upgrade breaks something and the newer version of sonobuoy is not able to retrieve the results from the test plugin (`e2e-tests` here).
As a consequence, the test result is always `unknown` and that breaks the CI.

That's why I left it at the same version as before since all the tests still work even with a new k8s version.

Reply:
Can you create an issue to track updating sonobuoy at some point in the future? I guess I'm OK with leaving it out of date for now, but I want to make sure we circle back and update it later before it blocks something critical.

Reply:
Issue created here: https://github.com/rancher/system-upgrade-controller/issues/186

Reply:
Upgrade done in https://github.com/rancher/system-upgrade-controller/pull/185/commits/100e67633a62e4d5d85dbbca314069567bb7c863